dpdispatcher/ssh_context.py

```diff
@@ -647,30 +648,45 @@ def download(
         file_list = []
         # for ii in job_dirs :
         for task in submission.belonging_tasks:
+            try:
+                remote_file_list = self.sftp.listdir(
```

Comment:
I think `listdir` only lists files under the root directory, doesn't it? If the file is `aa/bb`, `remote_file_list` only contains `aa` and cannot find `aa/bb`.

Reply:
Also, i/o operation in the remote server might be expensive, so I suggest only doing that when necessary (i.e. `*` is in the filenames).

Reply:
Sorry, it truly ignores `aa/bb` ,  I will fix it later.
And thanks for your suggestions, I will try to set taking `listdir` only when wildcards are in the filenames.

Reply:
(Bohrium HDFS OpenAPI Context remain to be modified)